lib/sinon/util/fake_xml_http_request.js

```diff
@@ -611,4 +611,4 @@
         makeApi(sinon);
     }
 
-})(typeof self !== "undefined" ? self : this);
+})(typeof global !== 'undefined' ? global : this);
```

Comment:
I think it's failing already due to #660, but I'll go ahead and open a PR to correct this.